src/frontend/main/src/views/Tabbed.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import { styled } from '@mui/system';
+import TabsUnstyled from '@mui/base/TabsUnstyled';
+import TabsListUnstyled from '@mui/base/TabsListUnstyled';
+import TabPanelUnstyled from '@mui/base/TabPanelUnstyled';
+import { buttonUnstyledClasses } from '@mui/base/ButtonUnstyled';
+import TabUnstyled, { tabUnstyledClasses } from '@mui/base/TabUnstyled';
+
+const blue = {
```

Comment:
These colors maybe belong to the MUI Theme instead of the view.

Reply:
new  dev view for use in profile page im thinking..   
good \<o>